pkg/piecestore/psserver/bucket_refresher.go

```diff
@@ -37,9 +37,7 @@ func newService(log *zap.Logger, interval time.Duration, rt *kademlia.RoutingTab
 }
 
 // Run runs the bucket refresher service
-func (service *refreshService) Run(ctx context.Context) (err error) {
-	defer mon.Task()(&ctx)(&err)
```

Comment:
why take out mon.Task?

Reply:
oh since the new version doesn't return an error is it not necessary?

Reply:
exactly